glean-core/src/metrics/mod.rs

```diff
@@ -121,6 +123,8 @@ pub enum Metric {
     Jwe(String),
     /// A rate metric. See [`RateMetric`] for more information.
     Rate(i32, i32),
+    /// TODO
```

Comment:
Sadly this can't be in alphabetical order, because the order of these enum items matter when serializing/deserializing. If we change the order we might end up breaking loading up old data files. See line 88.

Reply:
Gotcha, makes sense. When reviewing this through the GitHub UI, it had masked line 88 so I didn't see that.